packages/shared/src/components/search/SearchBarSuggestion.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import { Button, ButtonSize } from '../buttons/Button';
+import { AiIcon } from '../icons';
+import { IconSize } from '../Icon';
+
+export interface SearchBarSuggestionProps {
+  suggestion: string;
+  onClick: () => void;
+}
+
+export const SearchBarSuggestion = ({
+  suggestion,
+  onClick,
+}: SearchBarSuggestionProps): ReactElement => {
+  return (
+    <Button
+      className="btn-secondary border-theme-divider-tertiary typo-subhead text-theme-label-tertiary"
```

Comment:
It's not actually a border from design perspective, it's a tag, and for that matter a new one.
So for now this implementation is ok 🙏

Reply:
Oh, cool. Thanks for the clarification, maybe we will have to coordinate it with the design to probably create the right object in Sketch to avoid confusion as the current object tells a `Button`. Let's keep this then for the moment 🚢